spec/query_builder_spec.cr

```diff
@@ -29,6 +29,23 @@ describe Avram::QueryBuilder do
     query.statement.should eq "SELECT * FROM users"
   end
 
+  it "can reset where for a specific column" do
+    query = new_query
+      .where(Avram::Where::GreaterThan.new(:age, "18"))
+      .where(Avram::Where::LessThan.new(:age, "81"))
+      .where(Avram::Where::Equal.new(:name, "Pauline"))
+    query.args.should eq ["18", "81", "Pauline"]
+
+    query.reset_where("name")
+
+    query.statement.should eq "SELECT * FROM users WHERE age > $1 AND age < $2"
+    query.args.should eq ["18", "81"]
+
+    query.reset_where(:age)
+
+    query.args.should be_empty
```

Comment:
Could you add a statement test here to match what was done on line 41?

Reply:
That's what I mentioned on Gitter :joy: . The result of `where_sql` is cached, so the test would fail. You can't call `statement` twice because it will always return the first result.

Reply:
Ahhh! 👍